examples/react-list/src/friend.machine.ts

```diff
@@ -0,0 +1,56 @@
+import { assign } from 'xstate';
+import { createModel } from 'xstate/lib/model';
+
+const friendModel = createModel(
+  {
+    prevName: '',
+    name: ''
+  },
+  {
+    events: {
+      SET_NAME: (value: string) => ({ value }),
+      SAVE: () => ({}),
+      EDIT: () => ({}),
+      CANCEL: () => ({})
+    }
+  }
+);
+
+export const friendMachine = friendModel.createMachine({
+  initial: 'reading',
+  states: {
+    reading: {
+      tags: 'read',
+      on: {
+        EDIT: 'editing'
+      }
+    },
+    editing: {
+      tags: 'form',
+      on: {
+        SET_NAME: {
+          actions: assign({ name: (_, event) => event.value })
```

Comment:
```suggestion
          actions: friendModel.assign({ name: (_, event) => event.value })
```
Use assign from the created model.